specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-12-12/cosmos-db.json

```diff
@@ -6530,6 +6530,12 @@
     },
     "CreateUpdateOptions": {
       "type": "object",
+      "properties": {
+        "throughput": {
```

Comment:
Does case matter here?

Reply:
Yes, the requires "throughput" must be in lower case. Our service is case insensitive so it's fine.